jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -67,10 +67,11 @@
     private Boolean _persistent = null;
 
     private final int _send;
+    private final boolean _chunkEOFContent;
```

Comment:
OK, 9.4 doesn't yet do it, but it has been restructured in a way that would allow the chunked encoding to be used as a hint.  Would yo be able to use this if we added it to 9.4.1?